lib/macho/macho_file.rb

```diff
@@ -362,14 +366,26 @@ def rpaths
     #  file.change_rpath("/usr/lib", "/usr/local/lib")
     # @param old_path [String] the old runtime path
     # @param new_path [String] the new runtime path
+    # @param options [Hash]
+    # @return [void]
```

Comment:
Something went wrong here. (Note that there's already a `@return [void]` directly below this line.)